README.md

```diff
@@ -524,6 +525,8 @@ class YAxisExample extends React.PureComponent {
 | --- | --- | --- |
 | scale | `d3Scale.scaleLinear`| Should be the same as passed into the charts `yScale`, *or* d3Scale.scaleBand if used in conjunction with a horizontal BarChart |
 | svg | `{}` | supports all svg props an svg text normally supports |
+| spacingInner | 0.05 | Spacing between the labels |
+| spacingOuter | 0.05 | Spacing outside of the labels. Percentage of one label width |
```

Comment:
might need to add that this is only in conjunction with `scale=d3scale.scaleBand`